src/rustc-driver.md

```diff
@@ -7,7 +7,7 @@ using the interface defined in the [`rustc_interface`] crate.
 The `rustc_interface` crate provides external users with an (unstable) API
 for running code at particular times during the compilation process, allowing
 third parties to effectively use `rustc`'s internals as a library for
-analyzing a crate or emulating the compiler in-process (e.g. the RLS or rustdoc).
+analyzing a crate or emulating the compiler in-process (e.g. rustdoc).
```

Comment:
Heh, this should mention clippy and miri, just like all the other places that already list all the custom drivers (this chapter might need it more than most, since it looks like what I would link for "custom driver", though surprisingly it doesn't itself seem to use that term).

Reply:
@eddyb could that be a separate issue, so it doesn't block this PR